fedmsg.d/config.py

```diff
@@ -12,8 +12,11 @@
     'stomp_heartbeat': 10000,
     'estuary_updater.enabled': True,
     'estuary_updater.log_level': logging.INFO,
-    'estuary_updater.topics': [],
+    'estuary_updater.topics': [
+        '/queue/Consumer.client-estuary-updater.dev.VirtualTopic.eng.freshmaker.event.state.changed'
+    ],
     'estuary_updater.neo4j_url': 'bolt://neo4j:neo4j@localhost:7687',
+    'estuary_updater.koji_url': 'http://brewhub.engineering.redhat.com/brewhub',
```

Comment:
Optional: This can be obfuscated (e.g. "http://kojihub.domain.com/kojihub") to not expose our internal FQDN since it isn't required to be in the source. We will create our own config file in OpenShift when do the deployment to production that will contain the real values.